tests/test_wave_03.py

```diff
@@ -55,12 +55,15 @@ def test_friends_unique_movies_not_duplicated():
     # Assert
     assert len(friends_unique_movies) == 3
 
-    raise Exception("Test needs to be completed.")
+    
     # *************************************************************************************************
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
+    for films in amandas_data["watched"]:
+        for movies in friends_unique_movies:
+            assert movies["title"] != films["title"]
```

Comment:
Similarly here, how can we hardcode values specific to the test? If we have complex logic like this in our tests, we increase the risk that the testing logic is incorrect and we're not actually checking what we think we're checking.